qa/integration-tests/src/test/java/io/camunda/zeebe/it/backup/AzureBackupAcceptanceIT.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.backup;
+
+import com.azure.storage.blob.BlobServiceClient;
+import io.camunda.zeebe.backup.azure.AzureBackupConfig;
+import io.camunda.zeebe.backup.azure.AzureBackupStore;
+import io.camunda.zeebe.broker.system.configuration.backup.BackupStoreCfg.BackupStoreType;
+import io.camunda.zeebe.qa.util.cluster.TestApplication;
+import io.camunda.zeebe.qa.util.cluster.TestCluster;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneBroker;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import io.camunda.zeebe.qa.util.testcontainers.AzuriteContainer;
+import io.camunda.zeebe.test.util.testcontainers.ContainerLogsDumper;
+import java.util.Map;
+import org.agrona.CloseHelper;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+/**
+ * Acceptance tests for the backup management API. Tests here should interact with the backups
+ * primarily via the management API, and occasionally assert results on the configured backup store.
+ *
+ * <p>The tests run against a cluster of 2 brokers and 1 gateway, no embedded gateways, two
+ * partitions and replication factor of 1. This allows us to test that requests are correctly fanned
+ * out across the gateway, since each broker is guaranteed to be leader of a partition.
+ *
+ * <p>NOTE: this does not test the consistency of backups, nor that partition leaders correctly
+ * maintain consistency via checkpoint records. Other test suites should be set up for this.
+ */
```

Comment:
:wrench: At least part of that comment could be on the `BackupAcceptance` test interface